README.md

```diff
@@ -1,36 +1,17 @@
 # lightsaber [![Build Status](https://travis-ci.org/captn3m0/lightsaber.svg?branch=master)](https://travis-ci.org/captn3m0/lightsaber)
 
 Lightsaber is a simple DNS Redirect service. It offers 301/302 redirects for
-your domains. The configuration is kept public on this github repository itself.
+your domains. The configuration is stored in TXT records.
 
 ## Usage
 
 To add a DNS record, point your domain via a CNAME entry to `lightsaber.captnemo.in`.
 
-You can also have fallthrough domain redirects
-(thanks to @vivekprakash for the suggestion), by making the redirect a hash
-with a `root` key set to the url prefix.
+Then create a TXT record for the `_redirect` subdomain and use the following as the value:
 
-```yaml
-# This is a fallthrough redirect
-# so t.co/ev would redirect to twitter.com/ev
-t.co:
-  root: https://twitter.com
-# All goo.gl routes will forward to google.com, and not use fallthrough
-goo.gl: https://google.com
-```
+    v=lr1;https://google.com
```

Comment:
v=dr1;302;https//:google.com or do we want to fallback to 302/301, if not specified. Should we use key=value pairs to make parsing easier?

v=dr1;type=302;url=https://google.com or maybe a combination:

v=dr1;https://google.com would be interpreted as v=dr1;type=302;https://google.com and also as v=dr1;type=302;url=https://google.com